### ISSUE TITLE
Energy Recovery (Add on/Replace Component) 

### ISSUE THREAD

**Issue by khaddad** [canmet-energy/btap_tasks#274]
### Description
Write a decription of the issue. If it is a bug, please list all steps and attach a sample file, or sample ruby code that exhibits the bug. 

### Approach
After reviewing the issue, decribe your solution. If including git hyperlinks, ensure hyperlinks to existing code are to a version so it does not change with updates to git. 

### Testing Plan
Enter testing plan, if you are going to use an existing test or make a new one. 

### Waiting On
If this task is dependent on another task or NREL. Include notes Here. 

### Repositories Involved

https://github.com/NREL/openstudio-standards/tree/nrcan

### Select the Project Code you will be charging to 2018-2019

| Client 	|WSBE  	|Name   	|Manager   	
|----------	|---	|---	|---	
|Phylroy Lopez
|GI     |P-002930.001 |Better Evidence For Better Building Codes |Alex Ferguson

**Comment by phylroy** [comment]
Found this old gem in our DND project @mlubun @khaddad . I'll have to modify it to remove an existing erv if present...but not that hard... Do these performance characteristics work with what you are planning for costing? Should we offer different "pre-packages" of ERV types?
``` ruby
      def self.enable_erv(
          model,
          autosizeNominalSupplyAirFlowRate = true,
          setNominalSupplyAirFlowRate = nil,
          setHeatExchangerType = 'Plate', # 'Rotary' or 'Plate'
          setSensibleEffectivenessat100CoolingAirFlow = 0.76,
          setSensibleEffectivenessat75CoolingAirFlow  = 0.81,
          setLatentEffectiveness100Cooling = 0.68,
          setLatentEffectiveness75Cooling = 0.73,
          setSensibleEffectiveness100Heating = 0.76,
          setSensibleEffectiveness75Heating = 0.81,
          setLatentEffectiveness100Heating = 0.68,
          setLatentEffectiveness75Heating = 0.73,
          setSupplyAirOutletTemperatureControl = true,
          setFrostControlType = 'None', # 'None', 'ExhaustAirRecirculation','ExhaustOnly','MinimumExhaustTemperature'
          setThresholdTemperature  = 1.7,
          setInitialDefrostTimeFraction = 0,
          nominal_electric_power = nil,
          setEconomizerLockout = true
        )
        erv_array = Array.new()
        erv_array << model.getAirLoopHVACs().each do |air_loop|
          BTAP::Resources::HVAC::Plant::add_erv(
            model,
            air_loop,
            autosizeNominalSupplyAirFlowRate,
            setNominalSupplyAirFlowRate,
            setHeatExchangerType, # 'Rotary' or 'Plate'
            setSensibleEffectivenessat100CoolingAirFlow,
            setSensibleEffectivenessat75CoolingAirFlow,
            setLatentEffectiveness100Cooling,
            setLatentEffectiveness75Cooling,
            setSensibleEffectiveness100Heating,
            setSensibleEffectiveness75Heating,
            setLatentEffectiveness100Heating,
            setLatentEffectiveness75Heating,
            setSupplyAirOutletTemperatureControl,
            setFrostControlType, # 'None', 'ExhaustAirRecirculation','ExhaustOnly','MinimumExhaustTemperature'
            setThresholdTemperature,
            setInitialDefrostTimeFraction,
            nominal_electric_power,
            setEconomizerLockout)
        end
        return erv_array
      end
```

**Comment by phylroy** [comment]
Found also erv measure that PNNL wrote. https://bcl.nrel.gov/node/39440
